core/domain/feedback_jobs_one_off_test.py

```diff
@@ -125,3 +125,96 @@ def test_message_count(self):
         # Check if the quantities have the correct values.
         self.assertEqual(output[0][1]['message_count'], 1)
         self.assertEqual(output[0][1]['next_message_id'], 2)
+
+
+class FeedbackSubjectOneOffJobTest(test_utils.GenericTestBase):
+    """Tests for the one-off feedback subject update job."""
+    EXP_ID_1 = 'eid1'
+
+    EXPECTED_THREAD_DICT1 = {
+        'status': u'open',
+        'state_name': u'a_state_name',
+        'summary': u'a small summary',
+        'original_author_username': None,
+        'subject': u'(Feedback from a learner)'
+    }
+
+    EXPECTED_THREAD_DICT2 = {
+        'status': u'open',
+        'state_name': u'a_state_name',
+        'summary': u'a small summary',
+        'original_author_username': None,
+        'subject': u'Some subject'
+    }
+
+    EXPECTED_THREAD_DICT3 = {
+        'status': u'open',
+        'state_name': u'a_state_name',
+        'summary': u'It has to convert to a substring as it exceeds the character limit.', # pylint: disable=line-too-long
```

Comment:
No need for the pragma. Do something like this:

    'summary': (
        'It has to ... exceeds '
        'the character limit.')

Reply:
Done